src/resourcetiming-decompression.js

```diff
@@ -133,7 +143,9 @@
         }
 
         var initiatorType = parseInt(data[0], 10);
-        var timings = data.length > 1 ? data.substring(1).split(",") : [];
+        data = data.length > 1 ? data.split(SPECIAL_DATA_PREFIX + SPECIAL_DATA_SIZE_TYPE) : [];
```

Comment:
The way I understand it, dimensions are at the front of the compressed string and separated from the rest by a `|`,  `decompressResources` ignores them for now.
Sizes are part of each timing tuple.
e.g.: `*0dimensions|timing*1sizes|timing*1sizes...`

if a resource appears more than once a page, are there multiple RT entries? I tried loading an img twice in a page in FF and only get one entry.